packages/edit-widgets/src/index.js

```diff
@@ -40,6 +40,9 @@ export function initialize( id, settings ) {
  */
 export function customizerInitialize( id, settings ) {
 	registerCoreBlocks();
+	if ( process.env.GUTENBERG_PHASE === 2 ) {
+		__experimentalRegisterExperimentalCoreBlocks( settings );
```

Comment:
Why was this not here before? How was the legacy widget block being registered?

cc @jorgefilipecosta

Reply:
Initially it was being registered in `registerCoreBlocks` along with everything else. I moved all the experimental blocks out in my previous PR #16626 so we could enable/disable them more easily.

Reply:
Got it. Wouldn't that break the screen in the meantime though? If someone loads the Customizer, they won't have the legacy widget block registered right now?

Reply:
Yup, it's looking a bit broken now unless you have the setting enabled 😅